apps/gimbtonomy/LEDMatrix.cpp

```diff
@@ -0,0 +1,47 @@
+/* LED Matrix Source File */
+
+#include "LEDMatrix.h"
+
+
+//Data constructor
+//LEDMatrix :: LEDMatrix (void)
+//{
+    //initialize PWM pins - pins tbd
+    
+    //redPWM(/*redLED*/);
+    //greenPWM(/*greenLED*/);
+    //bluePWM(/*blueLED*/);
+//}
+
+//Set how often it flashes/sec
+void LEDMatrix :: setFrequency(uint8_t msgFrequency)
+{
+    redPWM.period(1/msgFrequency);
+    greenPWM.period(1/msgFrequency);
+    bluePWM.period(1/msgFrequency);
+}
+
+//Set brightness/amount (pulsewidth) of each colour
+uint8_t LEDMatrix :: setDutyCycle(uint8_t msgColour)
+{
+    return (1/255)*msgColour;
```

Comment:
This is integer division, it would always result in 0. Considering writing this in another way